src/TheResumeTheme/Recipes/TheResumeTheme.Tests.recipe.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
What is this needed for?

Reply:
This is not needed. Coming probably from an off track dev merge.